src/utils/localStorage.util.ts

```diff
@@ -0,0 +1,12 @@
+export class LocalStorage {
+    static getItem<T>(key: string): T | null {
```

Comment:
Insert btoa and atob on this case. For better comprehesion read this : https://developer.mozilla.org/en-US/docs/Web/API/Window/atob